src/main/antlr4/Manifold.g4

```diff
@@ -55,7 +55,7 @@ expression:
   | namespacedIdentifier # VariableReferenceExpression
   | expression '=' expression # AssignmentExpression
   | 'primitive' 'port' typevalue (':' tupleTypeValue)? # PrimitivePortDefinitionExpression
-  | 'primitive' 'node' functionTypeValue (':' tupleTypeValue)? # PrimitiveNodeDefinitionExpression
+  | 'primitive' 'node' functionTypeValue # PrimitiveNodeDefinitionExpression
```

Comment:
No, what it means is that attributes and ports are both wrapped into the `functionTypeValue`, not kept separate.